libs/sdk-backend-tiger/src/backend/workspace/catalog/datasetLoader.ts

```diff
@@ -120,19 +121,21 @@ function createDateDatasets(attributes: JsonApiAttributeOutList): ICatalogDateDa
 export async function loadAttributesAndDateDatasets(
     client: ITigerClient,
     workspaceId: string,
-    includeObjectWithTags: string[],
+    rsqlTagFilters: string[],
```

Comment:
Maybe this param should be `rsqlFilter` and string only?

Reply:
It can be, but I see some symmetry between this function and methods in factory.ts. I use same functions and `addRsqlFilterToParams` can be possibly extended with next RSQL filter params. Therefore I prefer the use array, but if you want to change it, I can do it.